revdep/problems.md

```diff
@@ -1 +1,24 @@
-*Wow, no problems at all. :)*
\ No newline at end of file
+# devtools
+
+<details>
+
+* Version: 2.4.3
+* GitHub: https://github.com/r-lib/devtools
+* Source code: https://github.com/cran/devtools
+* Date/Publication: 2021-11-30 09:00:02 UTC
+* Number of recursive dependencies: 113
+
+Run `revdep_details(, "devtools")` for more info
+
+</details>
+
+## Newly broken
+
+*   checking Rd cross-references ... WARNING
+    ```
+    Missing link or links in documentation object 'lint.Rd':
+      ‘[lintr:lint_file]{lintr::lint()}’
```

Comment:
I think we renamed this file (lint_file.Rd to lint.Rd); is there any way to prevent the update from breaking the doc link?

devtools source just uses `[lintr::lint()]`

Reply:
A `devtools::document()` against 3.0.0 would fix it. And we can provide an `@alias'. Should we?

Reply:
will `@alias` fix it? I'm thinking of the CRAN version of devtools. if we break them they'll need to release

Reply:
Yes, `@aliases lint_file` should fix it I think. Created a PR for this.
Is there a way to selectively re-run `revdepcheck()`?

Reply:
Done, Confirmed fixed.